packages/fxa-admin-panel/src/components/TableYHeaders/index.tsx

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { ReactElement } from 'react';
+import { HIDE_ROW } from '../../../constants';
+
+interface TableYHeadersProps {
+  header?: string;
+  testId?: string;
+  borderL?: boolean;
+  children:
+    | ReactElement<TableRowYHeaderProps>
+    | ReactElement<TableRowYHeaderProps>[];
+}
+
+interface TableRowYHeaderProps {
+  header: string | ReactElement;
```

Comment:
Hmm, if I take `string` out of the type, I get TS complaints unless I pass in a fragment. A fragment is more flexible, but also adds extra characters and possibly lines since most values are strings. I could change this though.

<img src="https://user-images.githubusercontent.com/13018240/200086447-bc8124fd-a5e5-4032-8382-f6f8fd34d29b.png" width="420px">

![image](https://user-images.githubusercontent.com/13018240/200086573-5ac8cc5d-9a4d-4378-9411-f8b01e85d482.png)

Reply:
@dschom Looking into this a little more, I'd like to actually change this to _just_ `string` for now, at least for the `header` prop. Every instance of this is using a `string` except for one I just changed, so I'd need to change every single instance to a React fragment, and then, that feels a little awkward since `{header}` goes in an `h3` element. I think when we need to use a React element there, that's the time to consider refactoring.